test/fixtures/rules/std_rule_cases/L053.yml

```diff
@@ -65,3 +65,30 @@ test_pass_begin_end_statement_brackets_tsql:
   configs:
     core:
       dialect: tsql
+
+test_fail_leading_trailing_whitespace:
+  # This previously caused the post-fix parse check to fail.
+  fail_str: |
+    (
+        SELECT
+            foo,
+            bar,
+            baz
+        FROM mycte2
+    );
+  # Yes, the formatting looks bad, but that's because we're only running L053
+  # here. In the real world, other rules will tidy up the formatting.
+  fix_str: "\n    SELECT\n        foo,\n        bar,\n        baz\n    FROM mycte2\n;\n"
```

Comment:
Why using `\n` when that's not what's used for the `fail_str`? Shouldn't it be consistent? Would also make the initial space look like "bad".

Reply:
It's because of a YAML limitation. The fix string has a blank line at the end, but the YAML parser doesn't pick it up; it assumes blank line means end of string. The `fail_str` doesn't have a blank line. Think I should change it? I prefer to use "normal" multi-line strings when possible, using quoted strings with `\n` or other escape sequences only when necessary or for readability. Happy to change the `fail_str` if you like, though!

Reply:
I'll go ahead and change `fail_str`.

Reply:
Done!